engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -34,6 +38,8 @@ class EngineJobExecutionActor(jobKey: BackendJobDescriptorKey,
   val jobTag = s"${workflowId.shortString}:${jobKey.call.fullyQualifiedName}:${jobKey.index.fromIndex}:${jobKey.attempt}"
   val tag = s"EJEA_$jobTag"
 
+  private var callCachingActorProps: Props = _
```

Comment:
`cacheHitCopyingActor`
Also this null makes me unhappy! Could it be `: Option[Props] = None`